src/store/main.js

```diff
@@ -10,7 +10,7 @@ export default {
   state: {
     loggedUser: null,
     refreshState: true,
-    drawer: !!Platform.is.desktop,
+    drawer: !!Platform.is.desktop || Screen.width >= 1024,
```

Comment:
Platform.is.ipad ?
(https://quasar.dev/options/platform-detection#Properties)

Reply:
Je pense que le soucis vient plus de la largeur que de la plateforme (sur une tablette d'une autre marque de dimension similaire je pense qu'il y a le meme soucis)

Reply:
Le problème va se poser sur pas mal de plateforme (je pense aux tablettes googles, ...). Donc il faut simplement avoir la transition < 1024 = menu burger ->  >= 1024 = un menu sur le côté